src/extension.ts

```diff
@@ -12,11 +12,12 @@ import {
 
 /* local */
 import { AnsiblePlaybookRunProvider } from './features/runner';
-
+import { getConflictingExtensions, showUninstallConflictsNotification } from './extensionConflicts';
 
 let client: LanguageClient;
 
 export function activate(context: ExtensionContext): void {
+
```

Comment:
Is this some autoformatting change? In Python, it's usually required not to have an empty line right after the function signature.

Reply:
Apparently this was not from eslint so I was able to remove it. Still, i think it would be preferable to keep formatting for the linter alone. If there is something it does not do,
we can look into reconfiguring it in a separate change.